scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -162,6 +163,16 @@ class Console(Harness):
 
     def configure(self, instance):
         super(Console, self).configure(instance)
+        if self.regex is None or len(self.regex) == 0:
+            self.state = "failed"
+            # Console Harness id has only TestSuite id without TestCase name suffix.
```

Comment:
I see that getting tc_name is copied from line 256 (in handle method), there the comment is with `TODO`, that suggests something should be changed / solved later. Maybe it could be worth to extract it to separate method.

Reply:
changed to a method, thank you.